kubemarine/cri/containerd.py

```diff
@@ -256,11 +256,12 @@ def configure_ctr_flags(group: NodeGroup) -> None:
         elif registry_auth.get('username'):
             options.append(f'--user {registry_auth["username"]}' +
                            f':{registry_auth["password"]}' if registry_auth.get("password") else '')
-        # Add hosts-dir, if it's presented
-        if config_path:
-            options.append(f'--hosts-dir {config_path}')
```

Comment:
Won't it be simpler if just iterate over `set().union(registry.get('mirrors', {}).keys(), registry.get('configs', {}).keys(), cluster.inventory.get("services", {}).get("cri", {}).get('containerdRegistriesConfig', {}))`, and return these lines? In this case we could avoid extra patch

Reply:
Discussed to maintain common flag for all registries through `*=...`